### DISCUSSION TITLE
Squash and merge commit message title to always be a PR title

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#6102] [Pull Requests] [answered]
When there's only one commit for the PR, it becomes the squash and merge commit message. It's really unobvious and due to inconsistency leads to accidental merge commits with messages like `init` and other.

See https://github.com/flutter/flutter/issues/88615

**Comment by LyleDavis** [comment]
This would be really handy for us too.

We use a squash and merge strategy for numerous reasons, and for the sake of sensible automated release notes I'd really like to be able to make the use of the PR titles the default for commit messages regardless of how many commits there are. Because of this, single commit PRs are a huge cause of bad commit messages in master.

**Comment by nt4f04uNd** [comment, marked as answer]
Looks like this is now configurable via new option https://github.blog/changelog/2022-05-11-default-to-pr-titles-for-squash-merge-commit-messages/
